tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = mypy,pycodestyle,ruff,pytest
+isolated_build = True
+labels =
+  venv = py312-{mypy,pycodestyle,ruff,pytest}, py3{6,7,8,9,10,11,12}-pytest
+  toxbox = py312-{mypy,pycodestyle,ruff,pytest}, py3{7,8,9,10,11,12}-pytest
```

Comment:
Is the py36 difference between venv and toxbox intended? (I suppose yes, just making sure)

Reply:
yes.  The `py36-pytest` scenario still works when I run it locally and it's my main go-to for testing, so I want to keep it there.